### PR TITLE
Initial version of Cloud Load Test Tasks

### PR DESCRIPTION
**Pull request by omeshp**
Two cloud load test build tasks.

### CODE REVIEW

File: Tasks/RunLoadTest/task.json [hunk 1]
```diff
@@ -0,0 +1,89 @@
...
+      "displayName": "Advanced",
+      "isExpanded": false
+    }
+  ],
+  "inputs": [
+    {
+      "name": "TestDrop",
+      "type": "filePath",
+      "label": "Load test files folder",
+      "defaultValue": "$(Build.SourcesDirectory)\\loadtest",
```
Comment (omeshp) [inline Tasks/RunLoadTest/task.json]: In RM there is no Build.SourcesDirectory, we need to remove such default values. We had done cleanup of common tasks between release and build where we removed such values. Build will have templates for these instead.

File: Tasks/RunLoadTest/task.json [hunk 2]
```diff
@@ -0,0 +1,89 @@
...
+    {
+      "name": "ThresholdLimit",
+      "type": "string",
+      "label": "Number of permissible threshold violations",      
+      "required": false,
+      "helpMarkDown": "Number of threshold violations above which the load test outcome is considered unsucessful."
+    },
+    {
+      "name": "VsoAccountUrl",
+      "type": "string",
```
Comment (bryanmacfarlane) [inline Tasks/RunLoadTest/task.json]: The user shouldn't have to supply the TFS account url - it's in the job message

File: Tasks/RunLoadTest/task.json [hunk 3]
```diff
@@ -0,0 +1,89 @@
...
+      "label": "VSO account uri",
+      "defaultValue": "$(System.TeamFoundationServerUri)",
+      "groupName":"advanced",
+      "required": true,
+      "helpMarkDown": "Visual Studio Online Account URL."
+    },
+    {
+      "name": "Username",
+      "type": "string",
+      "label": "Alternate credential username",
```
Comment (bryanmacfarlane) [inline Tasks/RunLoadTest/task.json]: The user shouldn't have to provide creds to first party services.

File: Tasks/RunLoadTest/task.json [hunk 4]
```diff
@@ -0,0 +1,89 @@
...
+  "author": "Microsoft Corporation",
+  "version": {
+    "Major": 1,
+    "Minor": 0,
+    "Patch": 1
+  },
+  "demands": [
+    "msbuild"
+  ],
+  "groups": [
```
Comment (bryanmacfarlane) [inline Tasks/RunLoadTest/task.json]: Doesn't this need a visibility or Preview?  We can't release tasks that take tfs urls and first party creds

File: Tasks/RunLoadTest/task.json (hunk 2 above)
Comment (ddixit) [inline Tasks/RunLoadTest/task.json]: As CLT is VSO only service, we need the URL and creds when running on-premises. There is no easy way to determine the CLT service endpoint, as location service REST APIs are not available. On a non-hybrid scenario, ideally we want to make use of PAT, which is pending work.

File: Tasks/RunLoadTest/task.json (hunk 4 above)
Comment (ddixit) [inline Tasks/RunLoadTest/task.json]: Added Preview visibility filter.

**Pull request merged**